stable/prometheus-operator/README.md

```diff
@@ -100,6 +100,9 @@ $ helm install --name my-release stable/prometheus-operator --set prometheusOper
 A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an
 incompatible breaking change needing manual actions.
 
+### Upgrading from 7.x.x to 8.x.x
```

Comment:
Could you be a little more explicit here? I think it's also worth mentioning that Prometheus needs an upgrade to support the rules as well, in case someone has pinned this at an earlier version:

### Upgrading from 7.x.x to 8.x.x
Due to new template functions being used in the rules in version `8.x.x` of the chart, an upgrade to Prometheus Operator and Prometheus is necessary in order to support them. First, upgrade to the latest version of `7.x.x`
```sh
helm upgrade <your-release-name> stable/prometheus-operator --version 7.4.0
```
Then upgrade to `8.x.x`
```sh
helm upgrade <your-release-name> stable/prometheus-operator 
```

Reply:
Sure thing. Resolved by https://github.com/helm/charts/pull/18375/commits/1de1a0841661edd6c1aa29c7f8310c86d35d0c6b